jkamz/Questioner
# #163454296 create comment

### What does this PR do?
This PR creates the endpoint for adding a comment

### Description of the tasks to be completed
create an endpoint for posting comments to given questions
### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/Questioner`

`$ cd Questioner`

Check out to the PR's branch
`$ git checkout ft-create-comment-163454296`

### What are the relevant Pivotal Tracker stories?
#### 163454296
#### https://www.pivotaltracker.com/story/show/163454296

### Screeenshots

![comment](https://user-images.githubusercontent.com/11965186/51692933-fbf0bd00-200e-11e9-9568-d83474623262.png)
